client/rpc-servers/src/lib.rs

```diff
@@ -137,17 +134,18 @@ mod inner {
 	/// **Note**: Only available if `not(target_os = "unknown")`.
 	pub fn start_http<M: pubsub::PubSubMetadata + Default + Unpin>(
 		addr: &std::net::SocketAddr,
-		thread_pool_size: Option<usize>,
 		cors: Option<&Vec<String>>,
 		io: RpcHandler<M>,
 		maybe_max_payload_mb: Option<usize>,
+		tokio_handle: tokio::runtime::Handle,
 	) -> io::Result<http::Server> {
 		let max_request_body_size = maybe_max_payload_mb
 			.map(|mb| mb.saturating_mul(MEGABYTE))
 			.unwrap_or(RPC_MAX_PAYLOAD_DEFAULT);
 
 		http::ServerBuilder::new(io)
-			.threads(thread_pool_size.unwrap_or(HTTP_THREADS))
+			.threads(1)
+			.event_loop_executor(tokio_handle)
```

Comment:
I read the docs and checked the code.

I'm almost sure, but can someone confirm that this is correct?

Reply:
/cc @tomusdrw but given https://docs.rs/jsonrpc-http-server/18.0.0/jsonrpc_http_server/struct.ServerBuilder.html#method.threads sounds right to me

Reply:
By correct I mean that we are still answering requests in parallel.

Reply:
yeah, I was referring to

> So it’s also possible to run a multi-threaded server by passing the default tokio::runtime executor to this builder and setting threads to 1.

Reply:
Yeah, my second comment was just to make my first one more clear xD

Reply:
Yup, I believe so too!